MapleServer2/Tools/HomeManager.cs

```diff
@@ -58,17 +58,17 @@ public List<Home> GetPlots(int mapId)
         private void SaveLoop()
         {
             Task.Run(async () =>
-              {
-                  while (true)
-                  {
-                      foreach (KeyValuePair<long, Home> kvp in HomeList)
-                      {
-                          DatabaseManager.Homes.Update(kvp.Value);
-                      }
-                      await Task.Delay(1000 * 60); // 1 minute
-                      //   await Task.Delay(1000 * 60 * 30); // 30 minutes
-                  }
-              });
+            {
+                while (true)
+                {
+                    foreach (KeyValuePair<long, Home> kvp in HomeList)
+                    {
+                        DatabaseManager.Homes.Update(kvp.Value);
+                    }
+                    await Task.Delay(1000 * 60); // 1 minute
+                    //   await Task.Delay(1000 * 60 * 30); // 30 minutes
```

Comment:
comment leftover

Reply:
its somewhat meant to stay like that but I can remove 🤷